consensus/ibft/ibft_test.go

```diff
@@ -752,7 +773,7 @@ func (t *mockTransition) Write(txn *types.Transaction) error {
 		}
 	}
 
-	t.transactionsWritten = append(t.transactionsWritten, txn)
+	t.successReceiptsWritten = append(t.successReceiptsWritten, txn)
```

Comment:
Why is this field `successReceiptsWritten` even relevant? We are testing the `writeTransactions()` method which doesn't contain any logic for writing receipts for successful transactions.

Reply:
You are right, we test everything with only expectedFailReceiptsWritten.

Reply:
fixed 14b2e7d2ac895d4423be57434df9e31d66942643

Reply:
I don't see this has changed, you still have this field `successReceiptsWritten`, just leave it as it was before